### PR TITLE
Pr42 adjust

### PR DESCRIPTION
**Pull request by brentp**
this is based off #42.

I'll let travis run so I can adjust as needed.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -18,23 +18,22 @@ install:
...
     cd ..
 before_script:
   - export PATH="nim-$nim_branch/bin${PATH:+:$PATH}"
+  - export NIM_LIB_PREFIX="$(pwd)/nim-$nim_branch"
 script:
-  - nim compile --verbosity:0 --run test/test
-  - |
-    for f in examples/*.nim; do
-      nim compile --verbosity:0 --hints:off "$f" || exit
-    done
```
Comment (oprypin) [inline .travis.yml]: It seems that examples are not being compiled anymore?

Comment (brentp) [reply .travis.yml]: I think this is now done via `nimble test` IIUC. 
I did not change it, but it reads:
```
task test, "Test":
  exec "nimble c --verbosity:0 -r -y test/test"
  for f in listFiles("examples"):
    if f[^4..^1] == ".nim": exec "nim compile --verbosity:0 --hints:off " & f
```

**Comment by oprypin** [comment]
OK, seems ready to merge then. I'll just check locally.

**Comment by oprypin** [comment]
Thanks!

**Comment by brentp** [comment]
will you tag a new release?

**Comment by oprypin** [comment]
Yes, after about a day. Hoping that a few people will try it out in their projects first

**Comment by brentp** [comment]
ok great. For my 0.02, I've checked it on [duphold](https://github.com/brentp/duphold) and [mosdepth](https://github.com/brentp/mosdepth) and it works fine.

**Pull request merged**